cardano_cli_helper.py

```diff
@@ -464,13 +467,14 @@ def generateDelegationCertificatePledge():
     getCardanoCliValue(command, '')
 
 
-def buildPoolAndDelegationCertTx(TxIn, TTL, amount='1000000', network='mainnet', era='babbage-era'):
+def buildPoolAndDelegationCertTx(utxos, TTL, amount='1000000', network='mainnet', era='babbage-era'):
```

Comment:
I applied the same logic as in `getRawTxSimple` and used every utxos instead of just the first one. Otherwise there was not enough funds.

Reply:
OK